great-idea/product.html

```diff
@@ -0,0 +1,138 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea!</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+  <body>  
+    <div class = "container">
+      <header>
+    <nav>
+  <a href ="file:///C:/Users/e0149196/Desktop/Personal%20Info/Lambda%20fullstack/Git/UI-III-Flexbox-master/UI-III-Flexbox-master/great-idea/services.html">Services</a>
+  <a href ="file:///C:/Users/e0149196/Desktop/Personal%20Info/Lambda%20fullstack/Git/UI-III-Flexbox-master/UI-III-Flexbox-master/great-idea/product.html">Product</a>
+      <a href ="#">Vision</a>
+          <a href ="#">Features</a>
+              <a href ="#">About</a>
+                  <a href ="#">Contact</a>
+  </nav>
+
+  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">

```

Comment:
Since there aren't any links back to the index page, you could wrap this image tag with anchor elements and have users taken back to the home page when they click on it.